openssl-sys/src/lib.rs

```diff
@@ -66,6 +64,47 @@ pub struct BIO_METHOD {
 unsafe impl Sync for BIO_METHOD {}
 
 #[repr(C)]
+pub struct RSA {
+    pad: c_int,
```

Comment:
We'll want all of these to be public since this crate just contains the raw FFI bindings.